STEERING-COMMITTEE.md

```diff
@@ -37,8 +38,6 @@ the way we run this committee, based on feedback from the community.
    disposition of those resources.
 1. Manage the Knative brand and decide which things can be called "Knative" and
```

Comment:
My hunch, though I am not a lawyer, is that we have created an unincorporated association with Google as the trustee of the Knative trademarks. That's generally an OK thing for the community, since we have these rules agreed to.

Reply:
You raise a valid question, I'm not sure I have the brain to answer it. I'll think it over, just want to explicitly ack it.

Reply:
While we're in here, can we add "Logo" into this bullet point since they control that too?